lib/public/init.js

```diff
@@ -27,6 +27,16 @@ module.exports = function (browserSync, name, pjson) {
 
         args.config.version = pjson.version;
 
+        /**
+         * Preserve the httpModule property's functions.
+         * the http2 module exports an object of functions and the merge function seems
+         * to want to destroy that, but if the base object is a function it seems fine
+         * TODO: find a better or more generic way to handle this
+         */
+        if(args.config.httpModule && !_.isFunction(args.config.httpModule)) {
+            args.config.httpModule = Object.assign(function() {}, args.config.httpModule);
+        }
```

Comment:
@shakyShane Still getting the same error.
`
[16:10:45] Starting 'serve'...  
[16:10:45] 'serve' errored after 204 μs  
[16:10:45] ReferenceError: http2 is not defined  
    at Gulp.<anonymous> (/Users/patrick/printfirm-docker/printfirm.com/gulpfile.js:25:21)  
    at module.exports (/Users/patrick/printfirm-docker/printfirm.com/node_modules/orchestrator/lib/runTask.js:34:7)  
    at Gulp.Orchestrator._runTask (/Users/patrick/printfirm-docker/printfirm.com/node_modules/orchestrator/index.js:273:3)  
    at Gulp.Orchestrator._runStep (/Users/patrick/printfirm-docker/printfirm.com/node_modules/orchestrator/index.js:214:10)  
    at Gulp.Orchestrator.start (/Users/patrick/printfirm-docker/printfirm.com/node_modules/orchestrator/index.js:134:8)  
    at /usr/local/lib/node_modules/gulp-cli/lib/versioned/^3.7.0/index.js:46:20  
    at _combinedTickCallback (internal/process/next_tick.js:67:7)  
    at process._tickCallback (internal/process/next_tick.js:98:9)  
    at Module.runMain (module.js:606:11)  
    at run (bootstrap_node.js:394:7)`